app/app.go

```diff
@@ -452,6 +467,30 @@ func NewApp(
 	return app
 }
 
+func inflationCalculator(ctx sdk.Context, minter minttypes.Minter, params minttypes.Params, bondedRatio sdk.Dec) sdk.Dec {
+	i0 := 100.0
+	t := ctx.BlockTime().Sub(genesisTime).Seconds() / (60 * 60 * 8766) // years passed
+	i := i0 * math.Pow(2, -t/tHalf)
+	inflation := sdk.NewDec(int64(i))
+
+	// min, max inflation based on a defined range parameter 'r'
+	// inflation range = [I(t) - I(t) * R, I(t) + I(t) * R]
+	r, err := sdk.NewDecFromStr("0.05") // let's say we allow 5% variance
+	if err != nil {
+		panic(err)
+	}
+	minInflation := minter.Inflation.Sub(minter.Inflation.Mul(r))
+	maxInflation := minter.Inflation.Add(minter.Inflation.Mul(r))
```

Comment:
I guess the issue I have is: does using `idealInflation + inflationRateChange` overly restrict the possible range?  If so then the entire range of acceptable inflation rates isn't available.

I think that we should use `idealInflation` to get `[max,min]`, and calculate `currentInflation = minter.Inflation + inflationRateChange`.  This way it will continue moving until an acceptable bonded ratio is found (in theory).

We also need to make a minimum `minInflation` - it can be `0` for now.

Reply:
> I think that we should use `idealInflation` to get `[max,min]`, and calculate `currentInflation = minter.Inflation + inflationRateChange`. This way it will continue moving until an acceptable bonded ratio is found (in theory).
> 
> We also need to make a minimum `minInflation` - it can be `0` for now.

Done